### PR TITLE
Fix broken features list in RTL languages

### LINKED ISSUE #2905: RTL on Mastodon translated /about page
Hi,
I would like to report a minor but not urgent little display problem on the /about instance page.
Note that strings are translated but the "ticked boxes" are still LTR instead of being RTL.

Arabic :
pic1
Hebrew :
pic2
Farsi :
pic3
On LTR languages seems there is no problem and is displayed "normally" :
pic4
pic5

What it *should* be like ?
pic6

Thank you for you attention.

### PR DESCRIPTION
**Pull request by ykzts**
Fixes #2905

### CODE REVIEW

**Comment by BoFFire** [comment]
That's correct :) Thank you @ykzts

File: app/javascript/styles/rtl.scss [hunk 1]
```diff
@@ -112,6 +112,16 @@ body.rtl {
     margin-right: 6px;
   }
 
+  .fa-ul {
+    margin-left: 0;
+    margin-left: 2.14285714em;
```
Comment (seefood) [inline app/javascript/styles/rtl.scss]: 2.14286132em might look a touch better.

Can you add the same for ar and he?

Thanks!

**Review by seefood — CHANGES_REQUESTED** [review]

File: app/javascript/styles/rtl.scss (hunk 1 above)
Comment (ykzts) [inline app/javascript/styles/rtl.scss]: This number is based on in [font-awesome.css](https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/css/font-awesome.css#L46). It is not something that can be changed by myself.

**Review by mjankowski — APPROVED** [review]

File: app/javascript/styles/rtl.scss (hunk 1 above)
Comment (seefood) [inline app/javascript/styles/rtl.scss]: ahhhh... so fa is font-awesome, not something specific to the Farsi language. OK :-)

**Pull request merged**